inst/components/scss/value_box.scss

```diff
@@ -1,8 +1,104 @@
+$bslib-value-box-enable-border: "auto" !default; // "auto" | "never" | "always"
+$bslib-value-box-enable-shadow: $enable-shadows !default;
+$bslib-value-box-horizontal-min-width: 300px;
+
+// CSS variables that start with --- (3 dashes instead of 2) are private vars
+// that are set by user arguments in the R calls. The CSS properties below only
+// apply to the default value box style; the standard way to theme individual
+// value boxes is with CSS classes on the value box container.
+// :root {
+//   --bslib-value-box-color:;
+//   --bslib-value-box-bg:;
+//   --bslib-value-box-border-color:;
+//   --bslib-value-box-border-width:;
+// }
+
 .bslib-value-box {
-  --bslib-value-box-separator-color: rgba(var(--bs-white-rgb, 255,255,255), 0.175);
+  @if ($bslib-value-box-enable-shadow) {
+    box-shadow: $box-shadow;
+  }
```

Comment:
Let's study what the `@include box-shadow()` is doing in Bootstrap (and generally how they think of it)

(e.g., do form controls get a box shadow when enabled (Worth testing on a white background?))

Reply:
Yeah, `$enable-shadows` turns on box shadows in many places. Instead of enabling that globally for value boxes in the shiny preset, I more directly set `$bslib-value-box-enable-shadow` in the Shiny preset. I have a feeling I'll be revisiting this, since the shadows are probably going to be something we want to include in specific contexts.